src/media-upload/index.js

```diff
@@ -76,7 +80,7 @@ import Constants from './constants';
  *     value={ id }
  * />
  */
-function MediaUpload( { onChange, value, ...otherProps } ) {
+function MediaUpload( { onChange, noticeOperations, noticeUI, value, ...otherProps } ) {
```

Comment:
Changing the order of these parameters potentially introduces a breaking change for blocks that use this component. Do you know if this is currently the case?

Reply:
In this proposal, the exported component is wrapped with the “withNotices” higher-order component that encapsulates the original component with the additional props “noticeOperations” and “noticeUI”. These properties would always have values, and the change in order wouldn’t affect the current usage of this component elsewhere. 🙂 

Additionally, I have published a beta version for testing purposes:
```bash 
npm i @sixa/wp-block-components@1.14.1-beta.0
```

Reply:
ah, yes, I see. Thank you for elaborating and for preparing a beta version, @mahdiyazdani